frontend/src/metabase/parameters/components/ParameterValueWidget.jsx

```diff
@@ -157,74 +159,95 @@ export default class ParameterValueWidget extends Component {
       isFullscreen,
       noReset,
       className,
+      dashboard,
     } = this.props;
     const { isFocused } = this.state;
     const hasValue = value != null;
+    const isDashParamWithoutMapping = isDashboardParameterWithoutMapping(
+      parameter,
+      dashboard,
+    );
+    const isDashParamWithoutMappingText = t`In order to use this filter it needs to be connected with a card on the dashboard.`;
```

Comment:
Pretty solid but I think we can make it even pithier.

```suggestion
    const isDashParamWithoutMappingText = t`This filter needs to be connected to a card.`;
```